starlight.spec

```diff
@@ -0,0 +1,19 @@
+### RPM external starlight r193
+Requires: clhep
+
+%define branch cms/%{realversion}
+%define github_user cms-externals
+Source0:git+http://github.com/%github_user/%{n}.git?obj=%{branch}/%{tag}&export=%{n}-%{realversion}&output=/%{n}-%{realversion}.tgz
+
+BuildRequires:	cmake
+
+%define keep_archives true
+
+%prep
+%setup -n %{n}/%{realversion}
+
+%build
```

Comment:
make is exist below %build
make %{makeprocess}
Should I set another line for make?